src/store/storeConfiguration.js

```diff
@@ -1,16 +1,20 @@
+/* eslint-disable comma-dangle */
 import { createStore, applyMiddleware } from 'redux';
 import reduxImmutableStateInvariant from 'redux-immutable-state-invariant';
 import thunk from 'redux-thunk';
 import { logger } from 'redux-logger';
 import { composeWithDevTools } from 'redux-devtools-extension';
-import rootReducer from '../reducers';
+import rootReducer from './rootReducer';
```

Comment:
This change was not necessary. The format written before, tell it to read the index file in the reducers folder.